pages/community/precice-workshop-2022.md

```diff
@@ -0,0 +1,68 @@
+---
+title: preCICE workshop 2022
+keywords: precice workshop 2022, event, events, workshop
+summary:
+permalink: precice-workshop-2022.html
+toc: false
+redirect_from: /preCICE2022/
+---
+
+<img class="img-responsive center-block" src="images/events/precice2022.svg" alt="preCICE Workshop banner" style="max-width: 500px; margin:auto;">
+
+The 3rd preCICE Workshop will be held virtually through [SimTech](https://www.simtech.uni-stuttgart.de/) and the [University of Stuttgart](https://www.uni-stuttgart.de/en/), from February 21-24, 2022. The workshop is a coming together of the preCICE community to share ideas, experiences and knowledge about using preCICE, and to learn from others in the process.
+
+[Registration](https://precice.org/precice-workshop-2022.html#registration) is now open. We look forward to welcoming back previous participants, and introducing more members to the preCICE community.
+
+Due to the uncertain times we are experiencing with the ongoing COVID-19 pandemic and the great success of the virtual workshop in February 2021, we are planning to host the workshop virtually again.
+
+If you have any questions regarding the workshop, please contact us at [`precice2022@ipvs.uni-stuttgart.de`](mailto:precice2022@ipvs.uni-stuttgart.de).
+
+## Program
+
+Coming soon.
+
+## Registration
+
+This is an academic, non-profit conference. However, we still have to collect a small registration fee.
+For early registrations (until January 17), the registration fee is
+
+* 50€ for members of universities and public research institutes and
+* 150€ for all other participants.
+
+Registration is still possible after this date, with a registration fee of
+
+* 100€ for members of universities and public research institutes and
+* 300€ for all other participants.
+
+Please use the [registration form](https://tagung.informatik-forum.org/preCICE2022/register) to register.
+
+For those who do not have the means to pay for the registration, you can write an email to us at [`precice2022@ipvs.uni-stuttgart.de`](mailto:precice2022@ipvs.uni-stuttgart.de), and we will evaluate each case individually. For selected cases we will supply a voucher to be used when registering that voids the required payment.
+
+## Call for contributions
+
+Please note that this is not a classical scientific conference, but a user and developer meeting. Contributions are very welcome, if they fit the purpose, but not mandatory.
+
+Possible contributions include (non-exclusively):
+<ul>
+<li>New adapters for community codes</li>
+<li>Coupling numerics</li>
+<li>Comparison of different mapping, coupling, and communication methods</li>
+<li>Coupling boundary conditions</li>
+<li>Multi-scale coupling</li>
+<li>Building, packaging, testing, documentation and other Research Software Engineering topics</li>
+</ul>
+
+Please use the [registration form](https://tagung.informatik-forum.org/preCICE2022/register) if you would like to submit a talk.
```

Comment:
That's the same link a third time :smile: 
I would click on all three and then be confused that they all lead to the same form.

Reply:
There is only one left now.